src/features/views/components/ViewDataTable/columnTypes/PersonFieldColumnType.tsx

```diff
@@ -0,0 +1,47 @@
+import { GridColDef } from '@mui/x-data-grid-pro';
+
+import { IColumnType } from '.';
+import { PersonFieldViewColumn, ZetkinViewColumn } from '../../types';
+import { EnumChoice } from 'utils/types/zetkin';
+
+type SimpleData = string | number | boolean | null;
+
+export default class PersonFieldColumnType
+  implements IColumnType<ZetkinViewColumn, SimpleData>
+{
+  cellToString(cell: SimpleData): string {
+    if (this.enumChoices) {
+      const choice = this.enumChoices.find((c) => c.key == cell);
+      return choice ? choice.label : '';
```

Comment:
As far as I can tell, this never happens when exporting a CSV, because this code is executed on the backend where `getColDef` will have never been called, and that's where the `this.enumChoices` state is (riskily) set.

Reply:
Fixed and verified that export works